src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2712,33 +2729,43 @@ public static <T> T handleVersion(String versionId, DsVersionHandler<T> hdl)
      * includeDeaccessioned default to false and checkPermsWhenDeaccessioned to false. Use it only when you are sure that the you don't need to work with
      * a deaccessioned dataset.
      */
-    private DatasetVersion getDatasetVersionOrDie(final DataverseRequest req, String versionNumber, final Dataset ds, UriInfo uriInfo, HttpHeaders headers) throws WrappedResponse {
+    private DatasetVersion getDatasetVersionOrDie(final DataverseRequest req, String versionNumber, final Dataset ds,
+            UriInfo uriInfo, HttpHeaders headers) throws WrappedResponse {
         //The checkPerms was added to check the permissions ONLY when the dataset is deaccessioned.
-        return getDatasetVersionOrDie(req, versionNumber, ds, uriInfo, headers, false, false);
+        boolean checkFilePerms = false;
+        boolean includeDeaccessioned = false;
```

Comment:
Same here, I would remove these variables and make the call simpler.

Reply:
I added the variable just to add context when reading the code so people don't have to go to the definition of the method to know what true/false does. (same from above, I can remove them but I like to give some context of what a parameter does without you having to dig into the method)